configure.ac

```diff
@@ -606,6 +608,8 @@ if test "$enable_cloud" = "yes"; then
     fi
 
     AC_MSG_RESULT([${enable_aclk}])
+else
```

Comment:
Why are you overwriting enable to no if it is not yes? This will interact very badly with the upcoming changes to the feature flag where the default will be a third option that must be preserved,

Reply:
There was problem that @Ferroin identified if he passed flag --enable-aclk and ACLK environment variable was not set it would still pick up that flag and define #define ENABLE_ACLK

It might be obsolete now....

Reply:
We've taken the env variable out completely and it is important that you do not overwrite the setting as it will persist.